app/scripts/metamask-controller.js

```diff
@@ -2184,18 +2142,14 @@ export default class MetamaskController extends EventEmitter {
       },
       version,
       // account mgmt
-      getAccounts: async (
-        { origin: innerOrigin },
-        { suppressUnauthorizedError = true } = {},
-      ) => {
+      getAccounts: async ({ origin: innerOrigin }) => {
+        // what?..
         if (innerOrigin === ORIGIN_METAMASK) {
           const selectedAddress =
             this.accountsController.getSelectedAccount().address;
           return selectedAddress ? [selectedAddress] : [];
         } else if (this.isUnlocked()) {
-          return await this.getPermittedAccounts(innerOrigin, {
-            suppressUnauthorizedError,
-          });
+          return await this.getPermittedAccounts(innerOrigin);
         }
         return []; // changing this is a breaking change
       },
```

Comment:
TODO: understand this better / verify it matters to us

Reply:
delete the comment afterwards

Reply:
AccountsController 🤷 ?

Reply:
I think we can safely ignore for now.